cfme/tests/automate/custom_button/test_infra_objects.py

```diff
@@ -96,6 +104,8 @@ def setup_obj(button_group, provider):
             obj = provider.appliance.provider_based_collection(provider).all()[0]
         elif obj_type == "TEMPLATE_IMAGE":
             obj = provider.appliance.collections.infra_templates.all()[0]
+        elif obj_type == "SWITCH":
+            obj = provider.appliance.collections.infra_switches.all()[0]
```

Comment:
Optional: thoughts on using a `dict` for these `if,elif` statements here?

Reply:
I will try but not with this PR.